packages/account/src/Sections/Profile/PersonalDetails/validation.ts

```diff
@@ -28,10 +31,14 @@ export const getPersonalDetailsInitialValues = (
         address_line_2: account_settings.address_line_2 ?? '',
         address_postcode: account_settings.address_postcode ?? '',
         address_state: '',
+        ...(isCountryCodeDropdownEnabled && { calling_country_code: selected_phone_code }),
+        ...(isCountryCodeDropdownEnabled && { is_carriers_available: is_carriers_supported }),
         date_of_birth: formatDate(account_settings.date_of_birth, 'YYYY-MM-DD'),
         first_name: account_settings.first_name,
         last_name: account_settings.last_name,
-        phone: `+${account_settings.phone?.replace(/\D/g, '')}`,
+        phone: isCountryCodeDropdownEnabled
+            ? account_settings.phone?.replace(/\D/g, '')
+            : `+${account_settings.phone?.replace(/\D/g, '')}`,
```

Comment:
In Ln:41 if `isCountryCodeDropdownEnabled` then Do we still need to replace non digits with ""?

Reply:
just in case BE returns spaces or any characters we should replace it with "". Because we don't know existing users might have spaces included in their phone number 🙏